lib/models/adlink.go

```diff
@@ -1,68 +1,58 @@
 package models
 
 import (
-	"github.com/beego/beego/v2/client/orm"
+	database "go-google-scraper-challenge/bootstrap"
 )
 
 type AdLink struct {
 	Base
 
-	Result          *Result  `orm:"rel(fk)"`
-	Type            string   `orm:"type(text)"`
-	Position        string   `orm:"type(text)"`
-	Link            string   `orm:"type(text)"`
+	ResultId int64   `gorm:"not null;"`
+	Result   *Result `gorm:"not null;foreignKey:ResultId"`
+	Type     string  `gorm:"not null;"`
+	Position string  `gorm:"not null;"`
+	Link     string  `gorm:"not null;"`
```

Comment:
```suggestion
	ResultID int64   `gorm:"not null;"`
	Type     string  `gorm:"not null;"`
	Position string  `gorm:"not null;"`
	Link     string  `gorm:"not null;"`
	
	Result   *Result `gorm:"not null;"`
```

Should we move association to another group?